src/Widgets/DataBreadcrumbWidget/DataBreadcrumbWidgetComponent.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  DataItem,
+  LinkTag,
+  provideComponent,
+  useData,
+  useDataLocator,
+} from 'scrivito'
+import { DataBreadcrumbWidget } from './DataBreadcrumbWidgetClass'
+import { ensureString } from '../../utils/ensureString'
+
+provideComponent(DataBreadcrumbWidget, ({ widget }) => {
+  const currentItem = useData().dataItem()
+  const parentAttributeName = useDataLocator(widget.get('parentData'))
+    .dataItemAttribute()
+    ?.attributeName()
+  const labelAttributeName = useDataLocator(widget.get('data'))
+    .dataItemAttribute()
+    ?.attributeName()
+
+  if (!currentItem) {
+    return <nav aria-label="breadcrumb" />
+  }
```

Comment:
FYI: This can be a one-liner:

```suggestion
  if (!currentItem) return <nav aria-label="breadcrumb" />
```